recipe/meta.yaml

```diff
@@ -18,6 +17,7 @@ source:
 
 build:
   number: 0
+  skip: true   # [abseil_cpp == '20211102.0']
```

Comment:
@hmaarrfk 
I'm not sure if you wanted this only for debugging purposes, or if it was intended to be merged. In any case, the skip is ineffectual - global skips don't work in a multi-output recipe, unfortunately.

Reply:
Actually, this is necessary because we don't have the new builds for 20211102.0 (and grpc builds for that abseil version are there with buildnum _0)...